### PR TITLE
Subnavigation implementation

### PR DESCRIPTION
**Pull request by DarinaBlazevica**

### CODE REVIEW

File: src/components/molecules/SubNavigation/SubNavigation.jsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
-const SubNavigation = (props) => {
+import "./SubNavigation.scss";
+import Arrow from "./angle-left-solid-black.svg";
```
Comment (CoobQ) [inline src/components/molecules/SubNavigation/SubNavigation.jsx]: Please move all SVG out of Component folder. You can create a separate folder as "Media" or something and hold images/svg there

Comment (DarinaBlazevica) [reply src/components/molecules/SubNavigation/SubNavigation.jsx]: Changed

File: src/components/molecules/Menu/Menu.scss [hunk 2]
```diff
@@ -4,7 +4,8 @@ $black: #000;
 $white: #fff;
 $hover: #d3d3d3;
 $subheading-color: #985e5b;
-$arial: Arial, sans-serif;
+$arial: Arial,
+sans-serif;
```
Comment (CoobQ) [inline src/components/molecules/Menu/Menu.scss]: Is there any reason to move this font name to a new row?

**Review by Jalal999 — APPROVED** [review]

File: src/components/molecules/Menu/Menu.scss [hunk 3]
```diff
@@ -82,6 +82,6 @@ $arial: Arial, sans-serif;
   outline: none;
 }
 
-input[type="text" i] {
+input[type="text"i] {
```
Comment (CoobQ) [inline src/components/molecules/Menu/Menu.scss]: For what "i" stands here for?

**Review by CoobQ — APPROVED** [review]

File: src/components/molecules/SubNavigation/SubNavigation.scss [hunk 4]
```diff
@@ -1,57 +1,64 @@
+$axis: "Axis";
```
Comment (ReinisZogots) [inline src/components/molecules/SubNavigation/SubNavigation.scss]: Shouldn't these be global?

File: src/components/molecules/Menu/Menu.scss (hunk 2 above)
Comment (DarinaBlazevica) [reply src/components/molecules/Menu/Menu.scss]: I have this every time when I format, will change

File: src/components/molecules/Menu/Menu.scss (hunk 3 above)
Comment (DarinaBlazevica) [reply src/components/molecules/Menu/Menu.scss]: Changed

File: src/components/molecules/SubNavigation/SubNavigation.scss (hunk 4 above)
Comment (DarinaBlazevica) [reply src/components/molecules/SubNavigation/SubNavigation.scss]: Solved in call with Reinis

**Pull request merged**